src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/CSharp/TypeAccessibilityCodeActionProvider.cs

```diff
@@ -77,9 +77,9 @@ internal class TypeAccessibilityCodeActionProvider : CSharpCodeActionProvider
             IEnumerable<RazorCodeAction> codeActions)
         {
             var diagnostics = context.Request.Context.Diagnostics.Where(diagnostic =>
-                diagnostic.Severity == DiagnosticSeverity.Error &&
-                diagnostic.Code?.IsString == true &&
-                s_supportedDiagnostics.Any(d => diagnostic.Code.Value.String.Equals(d, StringComparison.OrdinalIgnoreCase)));
+                diagnostic.Severity is not null && diagnostic.Severity.Value == DiagnosticSeverity.Error &&
+                diagnostic.Code is not null && diagnostic.Code.Value.TryGetSecond(out var str) &&
```

Comment:
You might not like it, so feel free to ignore, but I think this should work:
```suggestion
                diagnostic is { Severity: DiagnosticSeverity.Error, Code: { } code } &&
                code.TryGetSecond(out var str) &&
```

Reply:
I like it! `@davidwengier.PranavPoints++`.